test/qa/lib/daemon.py

```diff
@@ -188,3 +193,19 @@ def get_status_data() -> dict:
     colon_separated_pairs = (element.split(':') for element in lines)
     formatted_pairs = {(key.lower(), value.strip()) for key, value in colon_separated_pairs}
     return dict(formatted_pairs)
+
+def get_env() ->str:
```

Comment:
with the current implementation we are only going to be testing with prod configs during the release pipeline, do we not want to test prod configs with every run and only do dev environment config testing if there is something new to test?